pkg/utils/apiutil/apiutil.go

```diff
@@ -336,6 +336,22 @@ func ParseKey(name string, input map[string]any) ([]byte, string, error) {
 	return returned, rawKey, nil
 }
 
+// ParseHexKeys decodes hexadecimal keys to bytes if the format is "hex".
+func ParseHexKeys(format string, keys ...*string) error {
```

Comment:
Returning parsed key to user is better than returning result by input parameter implicitly.
```suggestion
func ParseHexKeys(format string, keys []string) ([]string, error) {
```

Reply:
want to reduce redundant code :(

Reply:
This suggestion aim to avoid to changing data in the input implicitly, and each update operation, I think, should re-assign to the old value. Let's see what other reviewers think.

Reply:
Perhaps there is no need to provide a function. Because of the multiple arguments and the error check which will make code more redundant.

Reply:
cc @JmPotato

Reply:
This involves converting between `[]byte` and `string`. I prefer a more performant approach; a benchmark may help.